Resources/Private/Partials/Application/Field/SelectCountry.html

```diff
@@ -4,7 +4,14 @@
 		<label><f:translate key="{translationKey}" /></label>
 	</div>
 	<div class="col-sm-9">
-		<s:form.select name="staticInfoTablesCountry" property="{fieldName}" staticInfoTable="country" optionLabelField="shortNameLocal" options="{}" class="form-control"/>
+		<f:if condition="{defaultCountry}">
+			<f:then>
+				<s:form.select name="staticInfoTablesCountry" property="{fieldName}" staticInfoTable="country" optionLabelField="shortNameLocal" options="{}" class="form-control" defaultOptionLabel="{defaultCountry.shortNameLocal}" defaultOptionValue="{defaultCountry.uid}"/>
```

Comment:
Is there a reason why this is not `value` instead? The country selected by default should be one of the available options anyways.

Reply:
@mbrodala because then the actual selected value is lost in case the user resumes an already started application.